privacy_evaluator/attacks/property_inference_attack.py

```diff
@@ -13,20 +14,30 @@
 from art.estimators.classification import TensorFlowV2Classifier
 import string
 
+# count of shadow training sets, must be even
+AMOUNT_SETS = 2
+# ratio and size for unbalanced data sets
+SIZE_SET = 1000
```

Comment:
I think we have to extract the size_set out of given data set

Reply:
Changed in new pull request so I resolved the conversation.